glide.lock

```diff
@@ -1,12 +1,10 @@
-hash: d5617480373ee580544dcaa0b3836aec445c98cefbdca0f0ce02b74428592997
-updated: 2017-09-06T14:32:24.658504132-07:00
+hash: 905eff0b6e3bd1dc48bb151f625c989c3468088728df1a357f58d00f7231305e
+updated: 2017-09-29T17:41:30.504994916+02:00
 imports:
 - name: github.com/andygrunwald/go-jira
   version: 9d1f282f93af41553ddb53b0116a8cdb75b4837a
 - name: github.com/armon/go-metrics
-  version: 97c69685293dce4c0a2d0b19535179bbc976e4d2
-- name: github.com/asaskevich/govalidator
-  version: 7b3beb6df3c42abd3509abfc3bcacc0fbfb7c877
+  version: 0a12dc6f6b9da6da644031a1b9b5a85478c5ee27
```

Comment:
This is the cause of the failure I mentioned above. The updated go-metrics requires Go 1.8 at least. It uses sort.Slice() introduced in Go 1.8.
Not sure why ci.centos does compile though...

Reply:
@alexeykazakov it compiles because the docker container in CI installs:

```
golang-bin             x86_64 1.8.3-1.el7                     base        47 M
 golang-src             noarch 1.8.3-1.el7                     base       4.8 M
```

Go to the build successful build log [here](https://ci.centos.org/job/devtools-fabric8-auth/211/consoleFull) and search for "`golang-1.8`" to check.

You check that yourself by running:
```
$ cd fabric8-auth
$ docker pull centos:7 # This is to refresh whatever centos:7 you've currently downloaded
$ make docker-start # Here you should see what gets installed
$ docker exec -it fabric8-auth-local-build go version
go version go1.8.3 linux/amd64
```